packages/cli/src/util/deploy/get-deployments-by-project-id.ts

```diff
@@ -45,17 +66,48 @@ export default async function getDeploymentsByProjectId(
     query.set('from', options.from.toString());
   }
 
-  const { deployments } = await client.fetch<Response>(
-    `/v4/now/deployments?${query}`
+  const { deployments: legacyDeployments } = await client.fetch<Response>(
+    `/v6/deployments?${query}`
   );
+
+  // we need to transform the old deployment shape to the new shape
```

Comment:
Can we instead use a newer API version to get the new Deployment type for all cases?

Reply:
This is the latest version: https://vercel.com/docs/rest-api#endpoints/deployments/list-deployments.

Reply:
So the latest version of `list-deployments` returns an old version of `Deployment` items? Can we ask Build and Deploy to look into creating a new version of the endpoint that responds with the latest version of `Deployment`?